.github/workflows/ci_e2e_cypress_base.yml

```diff
@@ -1,10 +1,9 @@
 name: e2e cypress BASE
 
 on:
-  push:

```

Comment:
Really good that we are also removing this now that we are basically moving to prod development